src/CarbonAware.DataSources/CarbonAware.DataSources.WattTime/src/Client/InternalsVisibleTo.cs

```diff
@@ -0,0 +1,4 @@
+using System.Runtime.CompilerServices;
```

Comment:
I would recommend to put this into the `csproj` file. No need for a new file.
See `GSF.CarbonAware.csproj` as an example

Reply:
will do!